allure-framework-modules/allure-webapi-module/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>allure-test-framework</artifactId>
+        <groupId>com.capgemini.ntc</groupId>
+        <version>2.0.0</version>
+    </parent>
+    <artifactId>allure-webapi-module</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>AllureTestFramework - WebApi - Module</name>
+    <description>Allure Test Framework WebApi supports:
+        TODO
+</description>
+    <url>https://github.com/devonfw/devonfw-testing</url>
+
+    <organization>
+        <name>Capgemini  - Nearshore Test Center - Poland</name>
+        <url>http://nsc.capgemini.com/nearshoretestcenter</url>
+    </organization>
+
+    <developers>
+        <developer>
+            <name>Lukasz Stefaniszyn</name>
+            <email>lukasz.stefaniszyn@capgemini.com</email>
+            <organization>Capgemini  - Nearshore Test Center - Poland</organization>
+            <organizationUrl>http://nsc.capgemini.com/nearshoretestcenter</organizationUrl>
+        </developer>
+    </developers>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/devonfw/devonfw-testing.git</connection>
+        <developerConnection>scm:git:ssh://github.com:devonfw/devonfw-testing.git</developerConnection>
+        <url>http://github.com/devonfw/devonfw-testing/tree/master</url>
+    </scm>
+
+    <properties>
+        <project.build.sourceEncoding>UTF8</project.build.sourceEncoding>
+
+        <allure-core-module.version>4.12.0.5-SNAPSHOT</allure-core-module.version>
+
+        <!-- rest-assured version -->
+        <io.rest-assured.version>3.0.2</io.rest-assured.version>
+        <org.jasypt.version>1.9.2</org.jasypt.version>
```

Comment:
If "jasypt" dependency is used in:
"webapi-module"  and in
"core-module". 
Therefore move please "jasypt" version to allure-framework-modules/pom.xml. And here only leave:
`
< dependency >

          < groupId>io.rest-assured</ groupId>

            < artifactId>json-schema-validator</ artifactId>

</ dependency>`